src/lib/ext-monitor.js

```diff
@@ -9,10 +9,10 @@ export default class ExtensionMonitor {
   extensionMapList = new Map([]);
 
   async getAllExtensions() {
+    const selfId = browser.runtime.id;
     const extensions = await browser.management.getAll();
-    const self = await browser.management.getSelf();
     return extensions.filter((extension) => {
-      return extension.type === 'extension' && extension.id !== self.id;
+      return extension.type === 'extension' && extension.id !== selfId;
```

Comment:
Just use `browser.runtime.id` directly. There is no need for storing it in a temporary variable.

Reply:
done :-)